release-notes/release-notes56.md

```diff
@@ -44,6 +44,10 @@ A new optional `name` field that sets a specific name to the format when it's be
 
 For information on the `name` field, see: [User interface - `style_formats`]({{ site.baseurl }}/configure/editor-appearance/#style_formats).
 
+### New card menu item for the autocompleter
+
+A new card menu item has been added to the autocompleter, allowing for greater customization of the autocompleter results displayed to the user. For information on customizing autocompleter results with `CardMenuItem`, see: [UI components - Autocompleter - `CardMenuItem`]({{ site.baseurl }}/ui-components/autocompleter/#cardmenuitem).
```

Comment:
The `For more information` part here should probably be on a new line, as that's what's done everywhere else. @tylerkelly13 can you confirm please?

Reply:
```suggestion
A new card menu item has been added to the autocompleter, allowing for greater customization of the autocompleter results displayed to the user.

For information on customizing autocompleter results with `CardMenuItem`, see: [UI components - Autocompleter - `CardMenuItem`]({{ site.baseurl }}/ui-components/autocompleter/#cardmenuitem).
```